vendor/bat-native-ads/src/bat/ads/internal/tokens/refill_unblinded_tokens/refill_unblinded_tokens.cc

```diff
@@ -107,7 +107,33 @@ void RefillUnblindedTokens::Refill() {
 
   nonce_ = "";
 
-  RequestSignedTokens();
+  InitiateRequestSignedTokens();
+}
+
+void RefillUnblindedTokens::InitiateRequestSignedTokens() {
```

Comment:
nit: could we rename `InitiateRequestSignedTokens` to `GetScheduledCaptcha`

Reply:
Fixed.